docs/source/client.rst

```diff
@@ -30,14 +30,12 @@ Below are examples of endpoints and how they map to API Client methods:
 +-----------------------------------+-------------------------------------------+
 | ``GET /workflows/1/executions``   | ``client.workflows.list_executions(1)``   |
 +-----------------------------------+-------------------------------------------+
-| ``PATCH /workflows/1``            | ``client.workflows.patch(1)``             |
+| ``PATCH /workflows/1``            | ``client.workflows.patch(1, ...)``        |
```

Comment:
I added `, ...` here since the patch function isn't really meaningful without some additional kwargs.  Also, I thought adding the additional kwargs could be confusing.